src/screens/FeedScreen/Header.js

```diff
@@ -187,7 +187,7 @@ const _renderAnonimity = ({
               buttonStyle={{position: 'absolute', right: 0, top: -8}}
               onPress={() => refSheet.current.open()}>
               <View style={{zIndex: 1000}}>
-                <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
+                <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `ElipsisIcon` component has been updated from `colors.blackgrey` to `COLORS.blackgrey`. This change is consistent with the update in the import statement.
```diff
- <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
+ <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->